t/compat.t

```diff
@@ -239,7 +239,7 @@ ok $mb, "Module::Build->new_from_context";
     );
 
     require ExtUtils::Install;
-    skip "Needs ExtUtils::Install 1.32 or later", 2 * @cases
+    skip "Needs ExtUtils::Install 1.32 or later", int(2 * @cases)
```

Comment:
I have no desire to maintain Module::Build also, sorry.
On May 10, 2016 20:31, "shadowcat-mst" <notifications@github.com> wrote:

> In t/compat.t
> <https://github.com/Perl-Toolchain-Gang/Module-Build/pull/68#discussion_r62726529>
> :
>
> > @@ -239,7 +239,7 @@ ok $mb, "Module::Build->new_from_context";
> >      );
> >
> >      require ExtUtils::Install;
> > -    skip "Needs ExtUtils::Install 1.32 or later", 2 * @cases
> > +    skip "Needs ExtUtils::Install 1.32 or later", int(2 * @cases)
>
> I suspect this change is going to be best left in your own tree, for the
> simple reason that "cperl adds type checking that doesn't DWIM yet" isn't
> really an argument for a change to a module targeting compatible perl
> implementations.
>
> I think a reasonable rule of thumb would be "if it works in pumpkin perl
> and Lehmann's stableperl but not in cperl, the workaround belongs in cperl,
> not here"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Perl-Toolchain-Gang/Module-Build/pull/68/files/5f6b3c3227b351d01d6a32b0dff0ba2788b2148e#r62726529>
>